citus.spec

```diff
@@ -44,7 +44,6 @@ make %{?_smp_mflags}
 %{__mkdir} -p %{buildroot}%{pginstdir}/doc/extension
 %{__cp} README.md %{buildroot}%{pginstdir}/doc/extension/README-%{sname}.md
 %{__cp} NOTICE %{buildroot}%{pginstdir}/doc/extension/NOTICE-%{sname}
-
```

Comment:
Let's keep the empty line that makes the file easier to read.

Reply:
Fixed as you suggested